osc-server/src/main/java/org/osc/core/broker/model/plugin/ApiFactoryServiceImpl.java

```diff
@@ -16,7 +16,7 @@
  *******************************************************************************/
 package org.osc.core.broker.model.plugin;
 
-import static java.util.Collections.emptyList;
+import static java.util.Collections.*;
```

Comment:
Is this star really necessary?

Reply:
for static imports our threshold is 3. I think in my workspace its set to 1. Will reconfigure.

https://github.com/opensecuritycontroller/community/blob/master/development/eclipse.md#code-style